deployments/helm/values.yaml

```diff
@@ -0,0 +1,179 @@
+image:
+  repository: grafana/beyla
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "main"
```

Comment:
I think this should be `latest` instead of `main`, as `main` is the development snapshot and `latest` is the last published version.